.github/workflows/run.yml

```
@@ -2,8 +2,8 @@ name: Build
 
 on:
   workflow_dispatch:
-  # schedule:
-  #   - cron: '0 * * * *'
+    schedule:
+      - cron: '5 * * * *'
```

indentation is off, should be a sibling to `workflow_dispatch`

---

```suggestion
  schedule:
    - cron: '5 * * * *'
```

@swalkinshaw is right 😊